src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -9,8 +9,8 @@
  */
 public class Tag {
 
-    public static final String MESSAGE_CONSTRAINTS = "Tags names should be alphanumeric";
-    public static final String VALIDATION_REGEX = "\\p{Alnum}+";
+    public static final String MESSAGE_CONSTRAINTS = "Tags names should not be empty";
+    public static final String VALIDATION_REGEX = ".+";
```

Comment:
Is there a difference between VALIDATION_REGEX = ".*" used in your previous PR and this ".+" ?

Reply:
also, the message constraints will never reach as you can put an empty tag

Reply:
basically, currently doesnt check if tags are empty or not. but tags should be able to be empty. wrong implementation but right outcome